ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -1611,3 +1611,54 @@ def test_tucker_tensorly(tol_norm_2, tol_max_abs, shape, ranks):
         ivy.max(ivy.abs(rec_svd - rec_random)) < tol_max_abs,
         "abs norm of difference between svd and random init too high",
     )
+
+
+# dot
+@st.composite
+def _generate_dot_dtype_and_arrays(draw):
+    input_dtype = [draw(st.sampled_from(["float32", "float64"]))]
+    print()
```

Comment:
```suggestion
    print()
```
let's remove this :sweat_smile: